mqtt/desired_state_client_test.go

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package mqtt
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	mocks "github.com/eclipse-kanto/update-manager/mqtt/mock"
+
+	gomock "github.com/golang/mock/gomock"
```

Comment:
The ```gomock``` alias is redundant. The ```mocks``` alias will also be redundant if the directory is renamed.